.github/workflows/changelog.yml

```diff
@@ -1,7 +1,7 @@
 name: Changelog
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
This unnecessarily [exposes secrets to these jobs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) and means that PRs updating these workflows cannot be evaluated until after they are merged making managing these files more difficult.

Reply:
that's fair, it makes workflow development difficult

Reply:
Reading into this a bit more the use of `pull_request_target` also provides a read/write `GITHUB_TOKEN` to the job. The [documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) warns that jobs using `pull_request_target` should not check out untrusted code (which occurs in the similarly modified `ci.yml` job). I'm going to close this PR.